### DISCUSSION TITLE
How to change media_player.volume_up/down step size?

### DISCUSSION THREAD

**Discussion by ministryofsillywalks** [music-assistant/support#2368] [Q&A] [answered]
Using AirPlay speakers i have an automation that turns up or down the volume depending on a button press on a zigbee remote.
However the step size always seems to be 5 so media_player.volume_up would take me from 20 to 25 which in some cases is not granular enough.
Is there any way to reduce the step size for these service calls?

**Comment by OzGav** [comment, marked as answer]
What about this? https://github.com/orgs/music-assistant/discussions/2330
